index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="./src/css/style.css">
+    <script src="./src/js/main.js" type="module"></script>
+    <title>The Dispatch</title>
+  </head>
+  <body>
+    <!-- main  -->
+    <nav id="blogHeader" class="blog__header navbar sticky-top container-fluid p-5 d-flex align-items-center justify-content-center">
+      <div class="blog__logo d-flex">
+        <a href="index.html" class="blog__header--title navbar-brand text-dark"><h1 class="blog__header--title">The Post Dispatch<img src="./assets/icons8-bugle-50.png" class="navbar__logo-img" alt="bugle icon"></h1></a>
+      </div>
+      <div class="header__form">
+        <form class="form-inline d-flex gap-3">
+          <input id="headerSearch" class="src__header form-control " type="search" placeholder="Search" aria-label="Search">
+          <button id="headerSearchBtn" class="btn btn-outline-success my-2 my-sm-0" type="button">Search</button>
+        </form>
+      </div>
+    </nav>
+    <!-- main content -->
+    <main id="mainContent" class="main__content  d-flex align-items-center pt-5">
+      <article id="postDisplay" class="d-flex container pt-5 flex-md-wrap justify-content-center overflow-auto gap-3">
+        <!-- post content -->
+      </article>
+      <!-- modal -->
+      <dialog id="modal" class="modal__dialog modal fade">
+        <!-- modal content -->
+        <article id="modalContent" class="modal-dialog modal-dialog-centered">
+          <div class="modal-content">
+            <header id="modalContent" class="modal-header">
```

Comment:
Error: Duplicate ID modalContent.

[From line 35, column 13; to line 35, column 59](https://validator.w3.org/nu/#l35c59)

          <header id="modalContent" class="modal-header">↩